### ISSUE TITLE
cp: cannot stat 'Synchronized-BatchNorm-PyTorch/sync_batchnorm': No such file or directory

### ISSUE THREAD

**Issue by XiaoluJiayou** [microsoft/Bringing-Old-Photos-Back-to-Life#208]
cp -rf Synchronized-BatchNorm-PyTorch/sync_batchnorm .

在终端运行这一句时，发生以上错误，不太懂最后一个空格加个点是什么意思。

**Comment by AZhurkin** [comment]
I have this issue too.

**Comment by renwu2019** [comment]
it means current directory

**Comment by Curious-Ray** [comment]
Manually do it.

First go to Synchronized-BatchNorm-PyTorch that you downloaded from github and copy the sync_batchnorm folder. Now put that folder in the following location:
1. Face_Enhancement/models/networks
2. Global/detection_models

This should fix the issue.
